src/lsp-server.ts

```diff
@@ -256,6 +256,13 @@ export class LspServer {
                     full: true,
                     range: true,
                 },
+                workspace: {
+                    fileOperations: {
+                        willRename: {
+                            filters: [{ pattern: { glob: '**/*.{ts,js}', matches: 'file' } }],
```

Comment:
Could also add `scheme: 'file'` since the server doesn't currently work for other unsaved files.

Reply:
I'm not sure what `scheme` means. I see this in spec:
```
	/**
	 * A Uri like `file` or `untitled`.
	 */
	scheme?: string;
```

Reply:
`scheme: file` would act as an additional filter so that the request should not triggered for files that are not saved on disk for example since server can't handle those.